### DISCUSSION TITLE
Alertmanager is unable to start

### DISCUSSION THREAD

**Discussion by gkalluru77** [learndevopsonline/learndevopsonline#1630] [Q&A]
IP Details:
prometheus : 3.92.166.181
front-end: 204.236.245.10
workstation: 18.209.172.17

Getting below error

3.92.166.181 | 172.31.42.111 | t3.small | null
[ root@prometheus ~ ]# ansible-pull -i localhost, -U https://github.com/gkalluru77/expense-ansible.git expense.yml -e role_name=prometheus
Starting Ansible Pull at 2023-11-23 10:08:19
/usr/local/bin/ansible-pull -i localhost, -U https://github.com/gkalluru77/expense-ansible.git expense.yml -e role_name=prometheus
[WARNING]: Could not match supplied host pattern, ignoring: ip-172-31-42-111
[WARNING]: Could not match supplied host pattern, ignoring:
ip-172-31-42-111.ec2.internal
localhost | CHANGED => {
    "after": "2d0e9228d64785ddc011e246f95f67697324c8c6",
    "ansible_facts": {
        "discovered_interpreter_python": "/usr/libexec/platform-python"
    },
    "before": "1fd2a9833e7e287469d685e47a1ad041bfe55e94",
    "changed": true,
    "remote_url_changed": false
}
[WARNING]: Could not match supplied host pattern, ignoring: ip-172-31-42-111
[WARNING]: Could not match supplied host pattern, ignoring:
ip-172-31-42-111.ec2.internal

PLAY [Expense Project setup] ***************************************************

TASK [Gathering Facts] *********************************************************
ok: [localhost]

TASK [prometheus : Copy Prometheus Configuration] ******************************
ok: [localhost]

TASK [prometheus : Copy Alert Rules] *******************************************
ok: [localhost]

TASK [prometheus : Restart Prometheus] *****************************************
changed: [localhost]

TASK [prometheus : Copy Alert Manager Config] **********************************
ok: [localhost]

TASK [prometheus : Restart alertmanager] ***************************************
changed: [localhost]

PLAY RECAP *********************************************************************
localhost                  : ok=6    changed=2    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0

3.92.166.181 | 172.31.42.111 | t3.small | null
[ root@prometheus ~ ]# systemctl status alertmanager
● alertmanager.service - AlertManager
   Loaded: loaded (/etc/systemd/system/alertmanager.service; enabled; vendor preset: disabled)
   Active: failed (Result: exit-code) since Thu 2023-11-23 10:08:26 UTC; 39s ago
  Process: 3286 ExecStart=/opt/alertmanager/alertmanager (code=exited, status=1/FAILURE)
 Main PID: 3286 (code=exited, status=1/FAILURE)

Nov 23 10:08:26 ip-172-31-42-111.ec2.internal systemd[1]: Started AlertManager.
Nov 23 10:08:26 ip-172-31-42-111.ec2.internal alertmanager[3286]: ts=2023-11-23T10:08:26.573Z caller=main.go:245 level=info msg="Starting Alertmanager" version="(version=0.26.0…361e295d8d)"
Nov 23 10:08:26 ip-172-31-42-111.ec2.internal alertmanager[3286]: ts=2023-11-23T10:08:26.573Z caller=main.go:246 level=info build_context="(go=go1.20.7, platform=linux/amd64, u…tags=netgo)"
Nov 23 10:08:26 ip-172-31-42-111.ec2.internal alertmanager[3286]: ts=2023-11-23T10:08:26.574Z caller=cluster.go:186 level=info component=cluster msg="setting advertise address …11 port=9094
Nov 23 10:08:26 ip-172-31-42-111.ec2.internal alertmanager[3286]: ts=2023-11-23T10:08:26.601Z caller=cluster.go:683 level=info component=cluster msg="Waiting for gossip to sett… interval=2s
Nov 23 10:08:26 ip-172-31-42-111.ec2.internal alertmanager[3286]: ts=2023-11-23T10:08:26.648Z caller=coordinator.go:113 level=info component=configuration msg="Loading configur…tmanager.yml
Nov 23 10:08:26 ip-172-31-42-111.ec2.internal alertmanager[3286]: ts=2023-11-23T10:08:26.648Z caller=coordinator.go:118 level=error component=configuration msg="Loading configu…sigv4.plain"
Nov 23 10:08:26 ip-172-31-42-111.ec2.internal alertmanager[3286]: ts=2023-11-23T10:08:26.649Z caller=cluster.go:692 level=info component=cluster msg="gossip not settled but con…=47.685251ms
Nov 23 10:08:26 ip-172-31-42-111.ec2.internal systemd[1]: alertmanager.service: Main process exited, code=exited, status=1/FAILURE
Nov 23 10:08:26 ip-172-31-42-111.ec2.internal systemd[1]: alertmanager.service: Failed with result 'exit-code'.
Hint: Some lines were ellipsized, use -l to show in full.

3.92.166.181 | 172.31.42.111 | t3.small | null
[ root@prometheus ~ ]# systemctl start alertmanager

3.92.166.181 | 172.31.42.111 | t3.small | null
[ root@prometheus ~ ]# systemctl status alertmanager
● alertmanager.service - AlertManager
   Loaded: loaded (/etc/systemd/system/alertmanager.service; enabled; vendor preset: disabled)
   Active: failed (Result: exit-code) since Thu 2023-11-23 10:09:33 UTC; 6s ago
  Process: 3306 ExecStart=/opt/alertmanager/alertmanager (code=exited, status=1/FAILURE)
 Main PID: 3306 (code=exited, status=1/FAILURE)

![image](https://github.com/learndevopsonline/learndevopsonline/assets/149259316/2d54adb8-724e-4559-8e8f-3249cfab08e7)

**Comment by sreedharm07** [comment]
@gkalluru77 
is this error solved in lab session or still exists ?
